noir-projects/noir-protocol-circuits/crates/types/src/delayed_public_mutable/scheduled_delay_change.nr

```diff
@@ -82,10 +82,9 @@ impl<let INITIAL_DELAY: u64> ScheduledDelayChange<INITIAL_DELAY> {
     }
 
     /// Returns the minimum delay before a value might mutate due to a scheduled change, from the perspective of some
-    /// historical timestamp (timestamp of a historical block). It only returns a meaningful value when called in
```

Comment:
Think this:

"(timestamp of a historical block)"

was more harmful than useful here.